### PR TITLE
r-styler whitespace

### PR DESCRIPTION
**Pull request by graemeblair**
prepping for CRAN

### CODE REVIEW

File: R/aaa.R [hunk 1]
```diff
@@ -33,97 +39,114 @@ dots_env_copy <- function(dots) {
...
 
 # Returns a new function(data) which calls FUN(data, dots)
-currydata <- function(FUN, dots, addDataArg = TRUE, strictDataParam = TRUE, cloneDots = TRUE) {
-  # heuristic to reuse deep clones
+currydata <-
+  function(FUN,
+           dots,
+           addDataArg = TRUE,
+           strictDataParam = TRUE,
+           cloneDots = TRUE) {
```
Comment (nfultz) [inline R/aaa.R]: This is bad style in general, if this was automated, please change the setting and keep things on one line if less than approx 100 chars, as in the R source code..

File: R/aaa.R [hunk 2]
```diff
@@ -33,97 +39,114 @@ dots_env_copy <- function(dots) {
...
+    strictDataParam = this$strictDataParam,
+    cloneDots = FALSE
+  ),
+  handler = handler,
+  dots = dots,
+  label = label,
+  step_type = this$step_type,
+  causal_type = this$causal_type,
+  call = match.call()
+  )
```
Comment (nfultz) [inline R/aaa.R:125]: Same, please change the ws back.

File: R/aaa.R [hunk 3]
```diff
@@ -33,97 +39,114 @@ dots_env_copy <- function(dots) {
...
-build_step <- function(curried_fn, handler, dots, label, step_type, causal_type, call){
-  structure(
+build_step <-
+  function(curried_fn,
+           handler,
+           dots,
+           label,
+           step_type,
+           causal_type,
+           call) {
```
Comment (nfultz) [inline R/aaa.R]: same.

File: tests/testthat/test-design-summary.R [hunk 4]
```diff
@@ -16,44 +16,43 @@ test_that("Basic design summary", {
 
   reveal_outcomes <- declare_reveal()
 
-  design <- my_population + 
-    my_potential_outcomes + 
-    my_sampling + 
-    my_estimand + 
-    declare_step(dplyr::mutate, q = 5) + 
-    my_assignment + 
-    reveal_outcomes +
```
Comment (nfultz) [inline tests/testthat/test-design-summary.R:25]: removing trailing spaces is good though.

File: R/aaa.R [hunk 5]
```diff
@@ -136,18 +159,22 @@ build_step <- function(curried_fn, handler, dots, label, step_type, causal_type,
...
-make_declarations <- function(default_handler, step_type, causal_type='dgp', default_label, strictDataParam=TRUE) {
-  
-  declaration <- declaration_template
-  
+make_declarations <-
+  function(default_handler,
+           step_type,
+           causal_type = "dgp",
+           default_label,
+           strictDataParam = TRUE) {
```
Comment (nfultz) [inline R/aaa.R]: should be all on same line.

File: R/declare_assignment.R [hunk 6]
```diff
@@ -53,56 +53,69 @@ declare_assignment <- make_declarations(assignment_handler, "assignment")
...
+validation_fn(assignment_handler) <- function(ret, dots, label) {
   declare_time_error_if_data(ret)
 
   dirty <- FALSE
 
   if (!"declaration" %in% names(dots)) {
     if ("blocks" %in% names(dots)) {
       if (class(f_rhs(dots[["blocks"]])) == "character") {
-        declare_time_error("Must provide the bare (unquoted) block variable name to blocks.", ret)
+        declare_time_error(
```
Comment (nfultz) [inline R/declare_assignment.R]: one line

File: R/declare_assignment.R [hunk 7]
```diff
@@ -121,21 +134,25 @@ validation_fn(assignment_handler) <-   function(ret, dots, label){
...
+        dots,
+        strictDataParam = attr(ret, "strictDataParam")
+      ),
+      handler = assignment_handler,
+      dots = dots,
+      label = label,
+      step_type = attr(ret, "step_type"),
+      causal_type = attr(ret, "causal_type"),
+      call = attr(ret, "call")
+    )
```
Comment (nfultz) [inline R/declare_assignment.R:134]: I give up.

**Review by nfultz — CHANGES_REQUESTED** [review]

**Comment by nfultz** [comment]
Make the whitespace look like this instead: https://github.com/tidyverse/ggplot2/blob/master/R/facet-wrap.r#L83

**Comment by acoppock** [comment]
this is all from goodpractice, which recommends less than 80 characters

**Comment by nfultz** [comment]
80 characters would be fine too (and it's a recommendation, nor a commandment), but splitting up function declarations across so many lines is pathologically bad style that noone does. The spirit of the 80 char max is to name things better, and not have thirty-character function names/arguments everywhere.

**Comment by graemeblair** [comment]
thanks for this, that makes sense. I undid the commits that were done manually (using CMD+shift+A). left the ones from styler that I don't think were objectionable on this.

**Comment by lorenzwalthert** [comment]
FYI: use `styler::style_text(..., strict = FALSE)` to make sure multiple white spaces are preserved: 
``` r
styler::style_text(c(
  "a   = 3",
  "abc = 4",
  "q   =3"
), strict = FALSE)
#> a   <- 3
#> abc <- 4
#> q   <- 3
```

Created on 2018-07-23 by the [reprex
package](http://reprex.tidyverse.org) (v0.2.0).

**Pull request merged**